packages/terra-date-time-picker/src/_TimeClarification.jsx

```diff
@@ -0,0 +1,171 @@
+};
+
+const defaultProps = {
+  isOpen: false,
+  isOffsetButtonHidden: false,
+  onDaylightSavingButtonClick: undefined,
+  onStandardTimeButtonClick: undefined,
+};
+
+const contextTypes = {
+  /* eslint-disable consistent-return */
+  intl: (context) => {
+    if (context.intl === undefined) {
+      return new Error('Please add locale prop to Base component to load translations');
+    }
+  },
+};
+
+class TimeClarification extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      isOpen: props.isOpen,
+      offsetDisplay: '',
+    };
+
+    this.handleOpenModal = this.handleOpenModal.bind(this);
+    this.handleCloseModal = this.handleCloseModal.bind(this);
+    this.handleDaylightSavingButtonClick = this.handleDaylightSavingButtonClick.bind(this);
+    this.handleStandardTimeButtonClick = this.handleStandardTimeButtonClick.bind(this);
+  }
+
+  componentWillReceiveProps(nextProps) {
+    if (nextProps === this.props) {
+      return;
+    }
+
+    this.setState({
+      isOpen: nextProps.isOpen,
+    });
+  }
+
+  handleOpenModal() {
+    this.setState({ isOpen: true });
+  }
+
+  handleCloseModal() {
+    this.setState({ isOpen: false });
+  }
+
+  handleDaylightSavingButtonClick(event) {
+    this.handleCloseModal();
+
+    this.setState({ offsetDisplay: DateTimeUtil.getDaylightSavingTZDisplay() });
+
+    if (this.props.onDaylightSavingButtonClick) {
+      this.props.onDaylightSavingButtonClick(event);
+    }
+  }
+
+  handleStandardTimeButtonClick(event) {
+    this.handleCloseModal();
+
+    this.setState({ offsetDisplay: DateTimeUtil.getStandardTZDisplay() });
+
+    if (this.props.onStandardTimeButtonClick) {
+      this.props.onStandardTimeButtonClick(event);
+    }
+  }
+
+  render() {
+    if (this.props.isOffsetButtonHidden) {
+      this.state.offsetDisplay = '';
+    }
+
+    const offsetButtonClassNames = cx([
+      'button-offset',
+      { 'button-offset-hidden': this.props.isOffsetButtonHidden || !this.state.offsetDisplay },
+    ]);
+
+    const intl = this.context.intl;
+    const title = intl.formatMessage({ id: 'Terra.dateTimePicker.timeClarification.title' });
+    const message = intl.formatMessage({ id: 'Terra.dateTimePicker.timeClarification.message' });
+    const daylightSavingButtonLabel = intl.formatMessage({ id: 'Terra.dateTimePicker.timeClarification.button.daylightSaving' });
+    const standardTimeButtonLabel = intl.formatMessage({ id: 'Terra.dateTimePicker.timeClarification.button.standardTime' });
+
+    return (
+      <div>
+        <Modal
+          classNameModal="time-clarification"
+          ariaLabel="Time Clarification"
+          isOpen={this.state.isOpen}
+          onRequestClose={this.handleCloseModal}
+          closeOnEsc={false}
+          closeOnOutsideClick={false}
+        >
+          <div>
+            <header className={cx('header')}>
+              {<h1 className={cx(['title'])}>{title}</h1>}
```

Comment:
nit: don't need the extra brackets around classname

Reply:
Updated: https://github.com/cerner/terra-core/pull/768/commits/3763caa7628bc29b961a755d2d760b627a28bbfa